index.js

```diff
@@ -0,0 +1,98 @@
+
+const url = "https://api.github.com/repos/Elena-Nam/Open-API-Project";
+fetch(url)
+  .then(response => response.json())
+  .then(data => console.log(data))
+  .catch((e) => console.log(e));
+
+
+
+// Coffee section //
+
+/* Iced coffee */
+// get data about the names of iced coffee //
+async function getIcedCoffeeData() {
```

Comment:
This call and the `getIcedCoffeeImages` API call are both querying the same endpoint with the same parameters. So, they two calls can be combined and the images/data can be parsed out of a single response.

This would allow you to put the coffee name with the coffee image.